roles/migrationcontroller/defaults/main.yml

```diff
@@ -27,8 +27,8 @@ hook_runner_repo: "{{ lookup( 'env', 'HOOK_RUNNER_REPO') }}"
 hook_runner_version: "{{ snapshot_tag | default(lookup( 'env', 'HOOK_RUNNER_TAG')) }}"
 hook_runner_image_fqin: "{{ hook_runner_image }}:{{ hook_runner_version }}"
 mig_controller_image: "{{ registry }}/{{ project }}/{{ mig_controller_repo }}"
-mig_controller_limits_cpu: "100m"
-mig_controller_limits_memory: "800Mi"
+mig_controller_limits_cpu: "1"
+mig_controller_limits_memory: "10Gi"
```

Comment:
I believe we may want to increase the memory requests along with the limits here.

Reply:
Worth noting that some nodes may have <10Gi of RAM total.

Reply:
I don't think that would prevent it from scheduling. It would just mean you wouldn't be able to reach the max limit. If we wanted to ensure a minimum we'd want to up the requests. If we wanted to ensure a minimum amount of resources it wold require increasing the requests. The default requests seems OK for small clusters like we test on. Unless we feel strongly that a much higher minimum is required in all or most cases I'm not sure we should raise those. We've had problems deploying OCS and CAM on the same cluster because the OCS default requests are pretty high for instance. Trying to strike some kind of balance, but my mind could probably be changed.

Reply:
+1. I wasn't able to dig up any specific recommendations on how to decide on the pairing of limit and request. Given our circumstance of low minimum requirements and very high maximum requirements based on the cluster we get deployed to, I think this makes sense.

Reply:
For the record, I wasn't suggesting we increase the request to 10gi, but I was wondering if it should scale inline with the limit. Right now it's 350m I think, to linearly scale would put us around 4Gi which seems high. I can't remember specifics of the pitfalls of setting requests too low.